mne/viz/_3d.py

```diff
@@ -790,3 +790,115 @@ def plot_sparse_source_estimates(src, stcs, colors=None, linewidth=2,
     surface.actor.property.shading = True
 
     return surface
+
+
+def plot_dipoles(dipoles, trans, subject, subjects_dir,
```

Comment:
use subjects_dir=None by default to be able to use os.environ

see get_subjects_dir function.